bin/blackbox_decrypt_file

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#
+# blackbox_decrypt_file -- Decrypt one or more blackbox files.
+#
+
+set -e
```

Comment:
Lines 7 through 13 aren't needed.

Reply:
I added `source "${0%/*}/_blackbox_common.sh"` for $BLACKBOX_HOME, similar to other commands.
The parameter check was added to be able to change the error message, since the error message in `blackbox_edit_start` mentions editing, but I can remove it if you prefer.

Reply:
Ah!  I see!  Both are excellent points!